test/functional/mining_basic.py

```
@@ -71,6 +74,45 @@ def mine_chain(self):
         self.connect_nodes(0, 1)
         self.connect_nodes(1, 0)
 
+    def test_blockmintxfee_parameter(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_blockmintxfee_parameter`

Add return type annotation: `None`

(ANN201)